src/components/secret-display/SecretDisplaySection.js

```diff
@@ -128,6 +135,34 @@ export default function SecretDisplaySection({
   const typeLabel = type === WalletTypes.privateKey ? 'key' : 'phrase';
 
   const { colors } = useTheme();
+
+  const renderStep = useCallback(() => {
+    if (isHideRecoveryPhrase) {
+      return (
+        <ColumnWithMargins align="center" justify="center">
+          <AuthenticationText>
+            Your account has been created with biometric data. To see the
```

Comment:
cc @jeremygoldbrg @christianbaroni @mikearndt @g-a-v-i-n for copy

Reply:
also, I'm pretty sure there is PR for i18n touching this